virtualization/virtualization.md

```diff
@@ -0,0 +1,56 @@
+# Virtualization in OpenShift
+
+Access design documentation specific to virtualization (kubevirt) features in OpenShift.
+
+- [Cluster Overview Dashboard](http://openshift.github.io/openshift-origin-design/virtualization/cluster-dashboard/cluster-dashboard)
```

Comment:
```suggestion
```

The whole `cluster-dashboard` folder should probably be moved to the `admin-perspective` folder instead of within `virtualization`. I added a comment to add the link to the `admin-perspective.md` file too.